resources/frontend/js/convertkit.js

```diff
@@ -181,13 +181,24 @@ function convertStoreSubscriberEmailAsIDInCookie( emailAddress ) {
 			if ( convertkit.debug ) {
 				console.log( response );
 			}
+
+			return response.json();
```

Comment:
Noticed that this was missing, and as a result would mean the `result` in the next `.then` block would be null.